src/test/java/org/apache/datasketches/sampling/EbppsSketchTest.java

```diff
@@ -22,8 +22,10 @@
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.assertFalse;
 import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.fail;
 
 import java.util.ArrayList;
+import java.util.Arrays;
```

Comment:
This is probably extraneous?

Reply:
Yes, and so is: import static org.testng.Assert.fail;
I will fix.